src/policy/immix/immixspace.rs

```diff
@@ -278,12 +280,13 @@ impl<VM: VMBinding> ImmixSpace<VM> {
         args: crate::policy::space::PlanCreateSpaceArgs<VM>,
         space_args: ImmixSpaceArgs,
     ) -> Self {
-        #[cfg(feature = "immix_non_moving")]
-        info!(
-            "Creating non-moving ImmixSpace: {}. Block size: 2^{}",
-            args.name,
-            Block::LOG_BYTES
-        );
+        if space_args.never_move_objects {
+            info!(
+                "Creating non-moving ImmixSpace: {}. Block size: 2^{}",
+                args.name,
+                Block::LOG_BYTES
+            );
```

Comment:
This existing `info!` log here was intended to warn the user that the user is forcing the ImmxSpace not to move objects, which may hurt the performance.  We can still print this log if the user overrides `never_move_objects` with `cfg!(feature = "immix_non_moving")`.  Maybe raise it to the `warn!` level.

But if the current `ImmixSpace` is the dedicated non-moving space, it may not be worth logging this at the `info!` level because it is a normal thing.